pkgs/by-name/li/litterbox/package.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, libressl, fetchzip, pkg-config, sqlite }:
+
+stdenv.mkDerivation rec {
+  pname = "litterbox";
+  version = "1.9";
+
+  src = fetchzip {
+    url = "https://git.causal.agency/litterbox/snapshot/litterbox-${version}.tar.gz";
+    sha256 = "sha256-w4qW7J5CKm+hXHsNNbl9roBslHD14JOe0Nj5WntETqM=";
+  };
+
+  buildInputs = [ libressl sqlite ];
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildFlags = [ "all" ];
+
+  meta = with lib; {
+    homepage = "https://code.causal.agency/june/litterbox";
+    description = "Simple TLS-only IRC logger";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Plus;
```
Just `licenses.gpl3` is deprecated